### DISCUSSION TITLE
Specify the default page/resource of a panel

### DISCUSSION THREAD

**Discussion by bilogic** [filamentphp/filament#8252] [Help]
1. It seems that when I load `/admin`, the first item of the navigation will be loaded by default
2. I have a panel at `/admin` where pages and resources are grouped by their functions, thus first item may not be suitable as the default
3. Is there a built-in way to easily specify another page/resource as the default?

**Comment by pboivin** [comment]
I think you can make a custom Panel class and override the `getUrl()` method:
https://github.com/filamentphp/filament/blob/3.x/packages/panels/src/Panel/Concerns/HasRoutes.php#L127

**Reply by bilogic** [reply]
Hmm, that method seems quite lengthy, I'm concerned overriding it will have unwanted side effects

**Reply by bilogic** [reply]
@pboivin 

It did not work, I replaced the `return` with my target URL, but it still got routed to the Dashboard. It only worked when I removed `->pages([Pages\Dashboard::class])` from my panel. Seems like there are several places intercepting behavior.
